src/safeposix/syscalls/fs_calls.rs

```diff
@@ -1573,6 +1573,54 @@ impl Cage {
         0 //success!
     }
 
+    
+     //------------------------------------FCHMOD SYSCALL------------------------------------
+
+    pub fn fchmod_syscall(&self, fd: i32, mode: u32) -> i32 {
+        if let Some(wrappedfd) = self.filedescriptortable.get(&fd) {
+            let wrappedclone = wrappedfd.clone();
+            drop(wrappedfd);
+            let filedesc_enum = wrappedclone.read();
+            match &*filedesc_enum {
+                File(normalfile_filedesc_obj) => {
```

Comment:
The indentation in this function is all sorts of weird, can you make it so things align properly.

Reply:
Fixed in new commit.